vocabularies/UI.xml

```diff
@@ -1071,8 +1115,22 @@ The trend is
           </Annotation>
         </Property>
         <Property Name="RequestAtLeast" Type="Collection(Edm.PropertyPath)" Nullable="false">
+          <Annotation Term="Common.PrimitivePropertyPath"/>
```

Comment:
Why is this restricted to primitive properties?

Reply:
@BerSie's wish

Reply:
I can't remember on having commented to the RequestAtLeast property annotation ...

Reply:
Sorry, my fault.